src/core/inlines.js

```diff
@@ -65,6 +67,19 @@ export function run(conf) {
           );
           // remember which ones were used
           conf.respecRFC2119[matched] = true;
+        } else if (conf.xref && matched.startsWith("{{{")) {
```

Comment:
Does inline parser really need to depend on conf.xref?

Reply:
Just kept it to have a safe side on `conf.xref` enabled or not. Removing this :slightly_smiling_face: